Tests/Unit/MSFT_xFirewall.Tests.ps1

```diff
@@ -149,6 +149,7 @@ try
                 }
             }
             Context 'Ensure is Present and the Firewall is Absent' {
+                Mock Get-FirewallRule
```

Comment:
Ok I'll play around with it let you know if I figured out something nice, I think I see some room for improvment but I'll have to play around with it a bit.